### DISCUSSION TITLE
How to delete oh-my-posh from powershell?

### DISCUSSION THREAD

**Discussion by wardward123** [JanDeDobbeleer/oh-my-posh#2752] [Troubleshoot]
Hello, I want to delete oh-my-posh from powershell but it says Uninstall-Package: No match was found for the specified search criteria and module names 'oh-my-posh'. when I type Uninstall-Module oh-my-posh -AllVersions

**Comment by JanDeDobbeleer** [comment]
@wardward123 oh-my-posh isn't a module anymore. You should be able to edit your `$PROFILE` and remove the `oh-my-posh init ...` line.

**Comment by JanDeDobbeleer** [comment]
@wardward123 oh-my-posh isn't a module anymore. You should be able to edit your `$PROFILE` and remove the `oh-my-posh init ...` line.

**Reply by techysiddhant** [reply]
winget install JanDeDobbeleer.OhMyPosh -s winget
replace install to uninstall

**Reply by wardward123** [reply]
it worked! thank you

**Reply by hassanad94** [reply]
As Well!

**Reply by deolrup-bh** [reply]
thanks

**Reply by denisiorodrigues** [reply]
It worked for me.

**Reply by laptuyet** [reply]
I encounter some error:
`Multiple versions of this package are installed. Either refine the search, pass the `--version` argument to select one, or pass the `--all-versions` flag to uninstall all of them.`

I'll try above command with more options `--all-versions` & it works.

**_winget uninstall JanDeDobbeleer.OhMyPosh -s winget --all-versions_**

**Comment by jassi1200** [comment]
![Screenshot 2024-07-31 121057](https://github.com/user-attachments/assets/04f756f1-95c0-4f4c-a749-4356c26d3451)
how to fix problum

**Reply by JanDeDobbeleer** [reply]
@jassi1200 which problem?

**Reply by jassi1200** [reply]
install problem

**Reply by jassi1200** [reply]
Please sir help

**Reply by JanDeDobbeleer** [reply]
Your question makes no sense, it says it's installed but as winget is concerned, you're on latest. Winget will always run behind as they have a manual review process.

**Reply by acctexperi** [reply]
Here is another way try this

- Get the path 

`$env:POSH_THEMES_PATH`

- Use unins000.exe to uninstall

`"C:\Users\Nitro-5\AppData\Local\Programs\oh-my-posh\unins000.exe"`

- Remove PS file from profile

`"C:\Users\Nitro-5\OneDrive\Documents\PowerShell\Microsoft.PowerShell_profile.ps1"`

**Comment by Sanjaya-Danushka** [comment]
i found the best way 
Install WinGet UI

Download from the official GitHub: [https://www.marticliment.com/unigetui/](https://www.marticliment.com/unigetui/)
(use direct download sometimes store link doesn't work )

Install it on your system.

Open WinGet UI

Click on the “Installed Packages” or “Apps” tab

It will list all software installed via winget.

Find “Oh My Posh” in the list

Click “Uninstall” next to it
remeber!  if it unstall failed.right click  package and unstall as adminstrator  
![image](https://github.com/user-attachments/assets/2d38bfcf-b0d5-4858-a721-bc288dd66bc5)
![image](https://github.com/user-attachments/assets/f634a8b8-ebb5-4d96-8601-764403fa8624)

**Reply by JanDeDobbeleer** [reply]
You can use Windows Add/Remove Program feature in the settings. It's listed there like any regular application.

**Reply by Sanjaya-Danushka** [reply]
that's true but if you install uniget you can update delete and install packages using one app
